cli/src/firebase_rtdb_service.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Service for making requests to the Firebase RTDB.
+"""
+
+from breakpoints_rtdb_service import BreakpointsRtdbService
+from exceptions import SilentlyExitError
+
+DEBUGGEE_NOT_FOUND_ERROR_MESSAGE = (
+    'Debuggee ID {debuggee_id} was not found.  Specify a debuggee ID found in '
+    'the result of the list_debuggees command.')
+
+
+class FirebaseRtdbService:
```

Comment:
Nit: I got somewhat confused between this service, the rest service, and the breakpoints service.

The main concerns I have are:
* FirebaseRtdbService and FirebaseRtdbRestService don't have obvious differences in responsibilities based on names.
* FirebaseRtdbService and BreakpointsRtdbService are both used for similar purposes.

My suggestion is to collapse the BreakpointsRtdbService and the FirebaseRtdbService into a single one (DebuggerRtdbService, for example).

Thoughts?

Reply:
Thanks for the comment!

Ah naming, always so hard :)

Agreed the current naming is confusing. Keeping the functionality in FirebaseRtdbRestService on its own makes sense, since in theory there could be  one that leverages the firebase cli instead for instance. (I realize you weren't advocating changing this, but figured it was worth mentioning). 

The reason for the breakpoints specific service was that the breakpoint related support is relatively large, so grouping it on its own make sense from that perspective, but I realiize as a result things aren't consistent and that potentially leads to more confusion than not. For now I think overall things would be better to do as you suggest, create a DebuggerRtdbService and have all the method grouped in there.

Reply:
This has now been addressed.